store/tikv/client_batch.go

```diff
@@ -430,7 +433,7 @@ func (a *batchConn) getClientAndSend(entries []*batchCommandsEntry, requests []*
 		}
 	}
 	if cli == nil {
-		logutil.BgLogger().Warn("no available connections", zap.String("target", target))
+		logutil.BgLogger().Info("no available connections", zap.String("target", target))
```

Comment:
I think Warn is better here.
In the common case such as the TiDB leader change, "no available connections" should not happen?

Reply:
All connections are not available maybe happen when TiKV is completely down and never recover.